api/models/Restaurant.js

```diff
@@ -0,0 +1,49 @@
+var mongoose = require('mongoose');
```

Comment:
nice to have it for reference purpose, can you add some comments on top so anyone comes here they know this is not the database data model?

Reply:
Sure, will do.